spec/griddler/email_spec.rb

```diff
@@ -784,3 +784,23 @@ def header_from_email(header)
     end
   end
 end
+
+describe Griddler::Email, 'extras param from adapter' do
+  it 'uses passes along adapter what the adapter sets' do
+    email = email_with_extras({ 'testing' => '123' })
+    expect(email.extras).to eq({ 'testing' => '123' })
+  end
+
+  it 'defaults to empty hash' do
+    email = email_with_extras(nil)
+    expect(email.extras).to eq({})
+  end
+
+  def email_with_extras(extras)
+    Griddler::Email.new({
+      to: ['hi@example.com'],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.